spec/system/page_objects/components/topic_list_header.rb

```diff
@@ -11,11 +11,11 @@ def topic_list_header
       end
 
       def has_bulk_select_button?
-        page.has_css?("#{TOPIC_LIST_HEADER_SELECTOR} button.bulk-select")
+        page.has_css?(".bulk-select")
       end
 
       def click_bulk_select_button
-        find("#{TOPIC_LIST_HEADER_SELECTOR} button.bulk-select").click
+        find(".bulk-select").click
       end
```

Comment:
These selectors were incorrect on mobile, the bulk buttons are not inside topic-list